ractor.rb

```diff
@@ -17,8 +22,8 @@
 # On CRuby (the default implementation), Global Virtual Machine Lock (GVL) is held per ractor, so
 # ractors are performed in parallel without locking each other.
 #
-# Instead of accessing the shared state, the data should be passed to and from ractors via
-# sending and receiving messages, thus making them _actors_ ("Ractor" stands for "Ruby Actor").
```

Comment:
Message passing is one important thing about actor-model, but I think the most important point is it is isolated with other actors. So I can't say it is the reason why it is an actor.